torch/_inductor/lowering.py

```diff
@@ -573,28 +574,7 @@ def to_dtype_bitcast(x: TensorBox, dtype: torch.dtype, *, copy=False):
     x_dtype = x.get_dtype()
     if x_dtype == dtype:
         return clone(x) if copy else x
-
-    def _get_primitive_bitwidth(dtype):
-        if dtype.is_floating_point:
-            return torch.finfo(dtype).bits
-        else:
-            return torch.iinfo(dtype).bits
-
-    src_bits = _get_primitive_bitwidth(x_dtype)
-    dst_bits = _get_primitive_bitwidth(dtype)
-    if src_bits != dst_bits:
-        # fallback to aten eager implementation for differing bitwidths
-        return fallback_handler(aten.view.dtype)(x, dtype)
```

Comment:
Still need to fallback for different bitwidth. we would need to change the size (along other changes) which we dont do right now. If we didnt have any skips in the dtype loop would have caught htis

Reply:
I see. get it back in https://github.com/pytorch/pytorch/pull/128883/commits/dfd1b9f77cda724c0a7a9124a3a249562fde50af